owncloudApp/src/main/java/com/owncloud/android/MainApp.kt

```diff
@@ -118,13 +131,64 @@ class MainApp : Application() {
 
                     } else {
                         ReleaseNotesActivity.runIfNeeded(activity)
+
+                        val pref = PreferenceManager.getDefaultSharedPreferences(appContext)
```

Comment:
We don't have currently a ViewModel for the MainApp, and I don't know if it would be suitable for this case... Anyway, this code is going to be predictably for 2 versions, then it won't be necessary anymore 👍